src/mlpack/core/dists/regression_distribution.cpp

```diff
@@ -65,6 +66,20 @@ double RegressionDistribution::Probability(const arma::vec& observation) const
   return err.Probability(observation(0)-fitted.t());
 }
 
+/**
+ * Evaluate probability density function of given observation.
```

Comment:
Since you already have this PR open, do you mind to fix the style issue here (missing spaces between the ``-``):

```
rf.Predict(observation.rows(1, observation.n_rows-1), fitted);
return err.Probability(observation(0)-fitted.t());
```